index.html

```diff
@@ -7,12 +7,18 @@
 </head>
 <body>
   <h1>Pixel Art!</h1>
+  <div id="swatch">
+    <div id="1"></div>
+    <div id="2"></div>
+    <div id="3"></div>
+  </div>
   <div class="controls">
     <form id="form">
-      <input type="text" id="color-field"></input>
```

Comment:
Whatchoo think you're doing, touching the HTML?! Use `addEventListener` in your JS instead.